### PR TITLE
update the changelog for m26

### PR DESCRIPTION
**Pull request by devoncarew**
@pq

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 ## 26.0
-- todo:
+- updates to support Android Studio 3.2 Beta
+- removes the Inspector's empty content message
+- setting log color from flutter log setting page
```
Comment (pq) [inline CHANGELOG.md]: nit: => 

    support setting log color from flutter log settings page

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+- support hiding/showing log categories (#2398)
+- add flutter log color settings page (#2394)
+- change the default for the open inspector setting
+- look for the emulator tool in the 'emulator/' directory first (#2383)
+- add feature filter by log level (#2380)
```
Comment (pq) [inline CHANGELOG.md]: nit, maybe:

    support filter by log level (#2380)

Comment (devoncarew) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (devoncarew) [reply CHANGELOG.md]: done

**Pull request merged**